widget/embedded/src/components/Quote/Quote.tsx

```diff
@@ -209,14 +247,13 @@ export function Quote(props: QuoteProps) {
   };
   const steps = getQuoteSteps(quote.result?.swaps ?? []);
   const numberOfSteps = steps.length;
-
   useLayoutEffect(() => {
-    if (expanded && quoteRef.current) {
+    if (autoScrollOnExpanding && expanded && quoteRef.current) {
```

Comment:
We have 3 problems here:

- auto scroll is not working when expanding the route. I think `autoScrollOnExpanding` logic is wrong.
- margin between alerts is wrong.
- When i click on change slippage on confirm swap, app crashes.

![Recording 2023-11-18 at 15 15 59](https://github.com/rango-exchange/rango-client/assets/99180934/56eea32c-488c-4a75-8cf6-bec0be9a4e6c)

Reply:
The third problem is still not fixed for me (app crashes)